quinn/tests/post_quantum.rs

```diff
@@ -0,0 +1,105 @@
+#![cfg(feature = "rustls-aws-lc-rs")]
+use proto::crypto::rustls::HandshakeData;
+use quinn::{
+    crypto::rustls::{QuicClientConfig, QuicServerConfig},
+    Endpoint,
+};
+use rustls::{
+    pki_types::{CertificateDer, PrivatePkcs8KeyDer},
+    NamedGroup,
+};
+
+use std::{
+    error::Error,
+    net::{Ipv4Addr, SocketAddr},
+    sync::Arc,
+};
```

Comment:
Please sort imports in `StdExternalCrate` blocks where both `quinn` and `proto` count as part of the crate.

Reply:
thanks that's cleaner, I've done it